### DISCUSSION TITLE
AN ACT for rendering a Written Memorandum necessary to the validity of Certain Promises and Engagements.

### DISCUSSION THREAD

**Discussion by maxw0206** [david-allison/manx-search-data#693] [_Archive] [answered]
Rather than Rob's English translation, I think we should use the English original as in Manx Sun June 10 1836.

**Comment by david-allison** [comment]
Hi Max,  The notes state that it's already taken from here (transcribed by Rob, taken from the original):

> Transcription; R. Teare, Translation from English version printed in the Manx Sun, June 10, 1836

https://manxcorpus.com/docs/Paber-Scruit-Ymmyrchagh?q=Written%20Memorandum

Is there anything else that should be done?

---

Refs if we need to do more:

https://www.imuseum.im/Olive/APA/IsleofMan/SharedView.Article.aspx?href=TMS%2F1836%2F06%2F10&id=Ar01000&sk=8A3A0CD4&viewMode=image

https://github.com/david-allison-1/manx-search-data/blob/5f9c51017ec96b267acdafa4f49fcb9125da2fde/OpenData/secular%20printed%20material%201750-1900/Slattys%20dy%20Yannoo%20Paber%20Scruit%20Ymmyrchagh/document.csv

**Comment by maxw0206** [comment]
Hi David

Well, the English version we have in the Corpus is not the English original in the Manx Sun, 

https://www.imuseum.im/Olive/APA/IsleofMan/SharedView.Article.aspx?href=TMS%2F1836%2F06%2F10 <https://www.imuseum.im/Olive/APA/IsleofMan/SharedView.Article.aspx?href=TMS%2F1836%2F06%2F10&id=Ar01000&sk=8A3A0CD4&viewMode=image> &id=Ar01000&sk=8A3A0CD4&viewMode=image

but rather, apparently, a re-translation of the Manx by Rob. 

 

I can edit the original English from Manx Sun and give it to you in Word to put in the Corpus.

 

Max

**Reply by david-allison** [reply]
Sounds good to me.

OCR is below

PS: Could you reply to discussions on GitHub rather than via email. Your email signature was included in the comment (I've removed it from the history).

Best,

David

```
ACT
For rendering a Written Memorandum necessary
to the validity of Certain Promises and Engage-
ments.

Promulgated at the Tynwald Hill on Tuesday last.

Wuerras, by an Act of Tynwald passed in
the Year One Thousand Seven Hundred and
Thirty eight, intituled “An Act for the Li-
“mitation of certain Actions and Claims for
“Debt, Trespass, and other things, for avoiding
“of Suits in Law,” it is amongst other things
enacted, “That all Actions or Plaints in nature
“of Actions of Accompts and upon the case,
“other than for Accom ts Current which con-
“cern the Trade of Merchandize between the
“Merchants and Tradcrs of this Is
“ Merchants and Traders, beyond Sea.
«Factors or Servants, all Actions cr Pl
“nature of Actions of Debt, grounded upon
“any Lending, Contract, cr Demand, without
“ Specialty, and all other Actions Transitory,
“ or Personal, whatscever which shall be sued
“or brought in any of the fempor al Courts of
“this Isle, or before any, gistrate of
“the same, after the makin: aw, shall
“be commenced and eectually prosecuted
“within Two Years next after the making of
“this Law, or within Three Years nex! after the
“cause of such Action, Plaint, or Suit, and at
“notime after.”
```

```
 

And whereas, various questions pave arisen
in Actions and Suits founded on simple con-
tract as to the proof and effect of Acknowledg-
ments and Picraises offered in lvidence for the
purpose of tuking Cases out of the operation
of the said Enactment, and it is expedient to
prevent such questions, and to make provision
for giving clfect to the said enactment,
the inter thereof
dutiful and Joy i
vernor, Council, Deemsters, and Keys of the
said Isle, do humbly beseech your Majesty that
it may be enacted, and be enacted by the

lent Majesty, by and with
consent of the Lieutenant-Go-
vernor, Council, Deemsters, and Keys of the
said Isle in Tynwald assembled, and by the au-
thority of the same, that in Actions of Debt or
upon the C: ounded upon any simple Con-
tract, no Acknowledgment or Promise by words
only shall be deemed sufficient evidence of a
new or continuing Centract whereby to take
any case ont of the operation of the said Enact-
ment, or to deprive any party of the benefit
thereof, unless such Acknowledgement or Pro-
```

```
  
  

or  Ad-
mini any Contractor, no such joint
Contractor, Executor, or Administrator shall
Jos ae benefit of the said Enactment so as to
be ¢ nle in respect, er by r On only, of
any written Acknowledgment or Promise made
d by any other or cthers of them.
t in Actions to be commenced
¢ joint Contractors, or Exe-

    

 

 

  

 

   

arge:

 

   

 

and sign!
Provided ziso, 1
inst two oF mic

    

           

    
  
  
  

      

cutors, oF rministrators, if it shall a
the trial or othe e that the Plainti
barred y the said recited Act as to on

 

more of such j
or Administr
to recover a:

Defendants bj

  

wT, Sere ney ertheless ae entitle
t any other or ers cf the
virtue of a newAcknowledgment
or Promise or otherwi Judgment may be
given and Costs allowed for the Plaintiff as to
such Defendant or Defendants against whom he
shall xecover, and for the otherDefeudant or
Defendants against the Plaintiff.

2.—And be it further enacted, that the s

recited Act and this Act shall be deemed and
```

```
taken to apply to the case of any Debt on
simple contract alleged by way of set-off on
the part of any Defendant either by Plea, No-
tice, or otherwise.

3.—And whereas, it is deemed necessary
for the prevention of fraudulent practices that
it be enacted, and be it enacted by the authority
aforesaid, that from and after the promulgation
of this Act, no Contract for the Sale of any
Goods, Weres, and Merchandizes, for the price
of Five Pounds, British, and upwards, shall be
allowed te be good except the Buyer shall aceept
part of the Goods so sold, and actually receive
the same, or give something in earnest to bind
the Bargaiv, or in part payment, or that some
Note or Memorandum, in writing, of the said
Bargain be made and signed by the parties to
be charged by such Contract, or their Agents
thereunto lawfully authorized.

4.—And be it further enacted, that the said
jast mentioned Enactment shall extend to all
Contracts for the Sale of Goods of the value
of Five Pounds, British, and upwards, notwith-
standing the Goods may be intended to he de-
livered at some future time, or may not at the
time of such Contract be actually made, pro-
cured, or provided, or fit or Ea for delivery,
cr some act may be requisite for the making
or completing thereof, or rendering the same fit
for delivery.
—And be it further enacted, that no Action
```

```
shall be maintained, whereby to charge any
person upon any promise made after full Ag
to pay any Debt contracted during infancy, or
spon any ratification after full Age. of any pro-
mise or simple contract made during Infancy,
unless such promise or ratification shall be made
by some Writing signed by the party to be
charged therewith.

6.—And be it further enacted, that no Action
1 be brought whereby to charge any Person
upon er by reason of any representation oF
assurance made or given concerning or relating
to the Character, Conduct, Credit, Ability,
Trade, or Dealings of any other Person, to
the intent or purpuse that such other Persoa
may obtain Credit, Money, or Goods thereupon,
unte: sentation or assurance be
made in signed by the Party to be
charged therewith.

7.—And be it further enacted, that this
Act shall apply to all cases of Engagements,
Indertakings, and Promises, made and entered
into by any Person as Bail, Pledge, or Security,
for the performance of any Agreement or Un-
lertaking, or for the paymen of auy Monies,
Goods, Wares, and Merchandizes, for the price
of Five Pounds, British, and upwards; and also
to all cases of Bail, Pledge,or Security gi
to the Ordinary or bis Spritual O: s for the
sufficient security of the Goods of deceased
ding to the luventory.
```

**Reply by david-allison** [reply]
Page 3 was a terrible scan. Improved it:

```
mise shali be made or contained by or in some
Writing to he signed by the Party chargeable
thereby; and that when there shall be two or
more joint Contractors, or Executors, or Ad-
ministrators of any Contractor, no such joint
Contractor, Executor, or Administrator shall
Jose the benefit of the said Enactmeut so as to
be chargeable in respect, or by reason only, of
any written Acknowledgment or Promise made
and signed by any other or others of them.
Provided also, that in Actions to be commenced
sainst two or more joint Contractors, or Exe-
itors, or Administrators, if it shall appear at
the trial or otherwise that the Plaintiff, though
barred by the said recited Act as to one or
more of such joint Contractors, or Execators,
or Admi: ators, shall nevertheless be entitled.
to recover ag t any other or others cf the
Defendants by virtue of a newAcknowledgment
or Promise or otherwise, Judgment may be
given and Costs allowed for the Plaintiff as to
such Defendant or Defendants against whom he
shall recover, and for the otherDefeudant or
Defendants against the Plaintiff.

2.—And be it further enacted, that the said
recited Act and this Act shall be deemed and
```

**Comment by maxw0206** [comment]
I’m working from the iMuseum OCR which is pretty good.

 

Thanks, anyway.

 

Max

**Reply by maxw0206** [reply]
The first paragraph alone of the Manx does not correspond to the first (WHEREAS) paragraph of the English.  Here is the English as in Manx Sun:

AN ACT
for rendering a Written Memorandum necessary to the validity of Certain Promises and Engagements.
Promulgated at the Tynwald Hill on Tuesday last.
WHEREAS, by an Act of Tynwald passed in the Year One Thousand Seven Hundred and Thirty eight, intituled “An Act for the Limitation of certain Actions and Claims for Debt, Trespass, and other things, for avoiding of Suits in Law,” it is amongst other things enacted, “That all Actions or Plaints in nature of Actions of Accompts and upon the case, other than for Accompts Current which concern the Trade of Merchandize between the Merchants and Traders of this Isle, and the Merchants and Traders, beyond Sea—their Factors or Servants, all Actions or Plaints in nature of Actions of Debt, grounded upon any Lending, Contract, or Demand, without Specialty, and all ether Actions Transitory, or Personal, whatsoever which shall be sued or brought in any of the Temporal Courts of this Isle, or before any Judge or Magistrate of  the same, after the making of this Law, shall be commenced and effectually prosecuted within Two Years next alter the making of this Law, or within Three Years next after the cause of such Action, Plaint, or Suit, and at no time after." 
And whereas, various questions have arisen in Actions and Suits founded on simple contract as to the proof and effect of Acknowledgments and Promises offered in Evidence for the purpose of taking Cases out of the operation of the said Enactment, and it is expedient to prevent such questions, and to make provision for giving effect to the said enactment, and to the intention thereof; We, your Majesty’s most dutiful and loyal subjects, the Lieutenant-Governor, Council, Deemsters, and Keys of the said Isle, do humbly beseech your Majesty that it may be enacted, and be it enacted by the King's Most Excellent Majesty, by and with the advice ana consent of the Lieutenant-Governor, Council, Deemsters, and Keys of the said Isle in Tynwald assembled, and by the authority of the same, that in Actions of Debt or upon the Case grounded upon any simple Contract, no Acknowledgment or Promise by words only shall be deemed sufficient evidence of a new or continuing Contract whereby to take any case out of the operation of the said Enactment, or to deprive any party of the benefit thereof, unless such Acknowledgment or Promise shall be made or contained by or in some Writing to be signed by the Party chargeable thereby : and that when there shall be two or more joint Contractors, or Executors, or Administrators of any Contractor, no such joint Contractor, Executor, or Administrator shall lose the benefit of the said Enactment so as to be chargeable in respect, or by reason only, of any written Acknowledgment or Promise made and signed by any other or others of them. Provided also, that in Actions to be commenced against two or more joint Contractors, or Executors, or Administrators, if it shall appear at the trial or otherwise that the Plaintiff, though barred by the said recited Act as to one or more of such joint Contractors, or Executors, or Administrators, shall nevertheless be entitled to recover against any other or others of the Defendants by virtue of a new Acknowledgment or Promise or otherwise, Judgment may be given and Costs allowed for the Plaintiff as to such Defendant or Defendants against whom he shall recover, and for the other Defendant or Defendants against the Plaintiff.
2.—And be it further enacted, that the said recited Act and this Act shall be deemed and taken to apply to the case of any Debt on simple contract alleged by way of set-off on the part of any Defendant either by Plea, Notice, or otherwise. 
3.—And whereas, it is deemed necessary for the prevention of fraudulent practices that it be enacted, and be it enacted by the authority aforesaid, that from and after the promulgation of this Act, no Contract for the Sale of any Goods, Wares, and Merchandizes, for the price of Five Pounds, British, and upwards, shall be allowed to be good except the Buyer shall accept part of the Goods so sold, and actually receive the same, or give something in earnest to bind the Bargain, or in part payment, or that some Note or Memorandum, in writing, of the said Bargain be made and signed by the parties to be charged by such Contract, or their Agents thereunto lawfully authorized. 
4.—And be it further enacted, that the said last mentioned Enactment shall extend to all Contracts for the Sale of Goods of the value of Five Pounds, British, and upwards, notwithstanding the Goods may be intended to be delivered at some future time, or may not at the time of such Contract be actually made, procured, or provided, or fit or ready for delivery, or some act may be. requisite for the making or completing thereof, or rendering the same fit for delivery. 
5.—And be it further enacted, that no Action shall be maintained, whereby to charge any person upon any promise made after full Age to pay any Debt contracted during infancy, or upon any ratification after full Age of any promise or simple contract made during Infancy, unless such promise or ratification shall be made by some Writing signed by the party to be charged therewith. 
6.—And be it further enacted, that no Action shall be brought whereby to charge any Person upon or by reason of any representation or assurance made or given concerning or relating to the Character, Conduct, Credit, Ability, Trade, or Dealings of any other Person, to the intent or purpose that such other Person may obtain Credit, Money, or Goods thereupon, unless such representation or assurance be made in Writing, signed by the Party to be charged therewith. 
7. —And be it further enacted, that this Act shall apply to all cases of Engagements, Undertakings, and Promises, made and entered into by any Person as Bail, Pledge, or Security, for the performance of any Agreement or Undertaking, or for the payment of any Monies, Goods, Wares, and Merchandizes, for the price of Five Pounds, British., and upwards; and also to all cases of Bail, Pledge, or Security given to the Ordinary or his Spiritual Officers for the sufficient security of the Goods of deceased Persons, according to the Inventory.

**Comment by david-allison** [comment, marked as answer]
Done in https://github.com/david-allison-1/manx-search-data/pull/697
